### ISSUE TITLE
add bats-assert and bats-file plugins to bats test suite

### ISSUE THREAD

**Issue by MichaelDimmitt** [exercism/bash#536]
I created a request in bats-core to add homebrew support for their plugins using homebrew taps.

https://github.com/bats-core/bats-core/issues/492

In the meantime, 
https://github.com/bats-core/bats-assert

Refers to 
https://github.com/ztombol/bats-docs

Which uses the homebrew tap from kaos
https://github.com/kaos/homebrew-shell

```sh
$ brew tap kaos/shell
```

```sh
$ brew install bats-assert
$ brew install bats-file
```

then in every test you have to load the files.
```bash
load '/opt/homebrew/lib/bats-support/load.bash'
load '/opt/homebrew/lib/bats-assert/load.bash'

@test 'refute_output()' {
  run echo 'want'
  refute_output 'want'
}
@test 'assert_equal()' {
  assert_equal 'have' 'hav'
}
```
<img width="923" alt="Screen Shot 2021-08-26 at 8 57 52 PM" src="https://user-images.githubusercontent.com/11463275/131054849-41422ac6-8976-4823-a28a-f5c7ee5ff632.png">

**Comment by glennj** [comment]
An additional note, the [list-ops test script](https://github.com/exercism/bash/blob/main/exercises/practice/list-ops/list_ops_test.sh) doesn't use `run`. While it would be nicer with `assert_equal`, it's OK without.

**Comment by kotp** [comment]
I think there is definitely value in keeping the locally running test suite simple.  It is kind of irritating to need to have node or a package system in order to install a system for testing bash.  A simple assertion program that can evaluate if there are matches is often fine for testing, and simple if it is the same language as being taught.  Less barrier to entry.

**Comment by glennj** [comment]
As I commented in #531, this has become more important for students coding in the online browser. Having the only test feedback as "the output failed" is no good.

What I did in the Tcl track is put a "testHelpers" file in every exercise, and the "<slug>.test" file will source it. That keeps the test file clean, but also makes the extra functionality available.

I propose to do the same thing for bats-assert: the source files for bats-assert (and the required bats-support) are small, so the can just be concatenated into a single file to be sourced.

A draft PR is forthcoming soon.
